hw/dv/tools/vcs/cover.cfg

```diff
@@ -8,6 +8,7 @@
 +tree tb.dut
 -module pins_if     // DV construct.
 -module clk_rst_if  // DV construct.
+-moduletree prim_alert_sender  // prim_alert_sender is verified in FPV.
```

Comment:
Add this to the `begin tgl` section below as well - we should ensure toggle coverage on its ports. 

This is a general argument - any sub-module that is black boxed from coverage collection must have its ports analysed for toggle.

Reply:
That is a good point, thanks i will add that.

Reply:
Should this be `prim_alert_sender#(*)`? May need to check with AE.

Reply:
I think this is sufficient, in otp_ctrl's case: both alert0 and alert1 are excluded:
![image](https://user-images.githubusercontent.com/11466553/119025274-d367fc00-b958-11eb-9d2d-e3503853f091.png)

Reply:
I think it is fine because all the alert_sender's module name is `prim_alert_sender`. Only instance names are different as they are: alert_sender[0], etc. So I think we are okay here

Reply:
Can you check if OTP control instantiates `prim_alert_sender` with different parameters than the default values in prim/rtl area?

Reply:
otp_code: https://github.com/lowRISC/opentitan/blob/master/hw/ip/otp_ctrl/rtl/otp_ctrl.sv#L479
```
  for (genvar k = 0; k < NumAlerts; k++) begin : gen_alert_tx
    prim_alert_sender #(
      .AsyncOn(AlertAsyncOn[k]),
      .IsFatal(1)
    ) u_prim_alert_sender (
      .clk_i,
      .rst_ni,
      .alert_test_i  ( alert_test[k] ),
      .alert_req_i   ( alerts[k]     ),
      .alert_ack_o   (               ),
      .alert_state_o (               ),
      .alert_rx_i    ( alert_rx_i[k] ),
      .alert_tx_o    ( alert_tx_o[k] )
    );
  end
```
Alert_sender module: https://github.com/lowRISC/opentitan/blob/master/hw/ip/prim/rtl/prim_alert_sender.sv

looks like OTP instantiates with different parameters :)